packages/vue/src/components/authenticator.vue

```diff
@@ -221,54 +228,56 @@ watch(
             @click="send('SIGN_UP')"
           />
         </base-two-tabs>
-        <sign-in
-          v-if="actorState?.matches('signIn')"
-          @sign-in-submit="onSignInSubmitI"
-          ref="signInComponent"
-        >
-          <template #signInSlotI>
-            <slot name="sign-in"></slot>
-          </template>
-
-          <template
-            #form="{ info, onSignInSubmit, onForgotPasswordClicked, onInput }"
+        <div v-if="hasTabs()" data-amplify-router-content>
```

Comment:
GitHub is showing a lot of spacing changes below... but the only code change is adding this `div` around it.